<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
	background-color: #3a3a3a;
	color: #fff;
	selection-background-color: #b78620;
	selection-color: #000;

}

/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;

}


/*-----QHeaderView-----*/
QHeaderView::section
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
    color: #fff;
    text-align: left;
	padding: 4px;
	
}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
    color: #fff;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


QTableCornerButton::section
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(60, 60, 60, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #000;
    color: #fff;

}

/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 2px;
	padding: 3px;
	padding-left: 5px;

}


/*-----QPlainTExtEdit-----*/
QPlainTextEdit
{
	background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	border-radius: 2px;
	padding: 3px;
	padding-left: 5px;

}


/*-----QToolButton-----*/
QToolButton 
{
	background-color: transparent;
	color: #717072;
	padding: 5px;
	padding-left: 8px;
	padding-right: 8px;
	margin-left: 1px;
	border: 1px solid #808080;

}


QToolButton:hover
{
	background-color: rgba(183, 134, 32, 20%);
	border: 1px solid #b78620;
	color: #fff;
	
}


QToolButton:pressed
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #b78620;

}


QToolButton:checked
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
	border: 1px solid #222;
}

/*-----QComboBox-----*/
QComboBox
{
    background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(84, 84, 84, 255),stop:1 rgba(59, 59, 59, 255));
    border: 1px solid #000;
    padding-left: 6px;
    color: #ffffff;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QComboBox:on
{
    background-color: #b78620;
	color: #000;

}


QComboBox QAbstractItemView
{
    background-color: #383838;
    color: #ffffff;
    border: 1px solid black;
    selection-background-color: #b78620;
    outline: 0;

}


QComboBox::drop-down
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(57, 57, 57, 255),stop:1 rgba(50, 50, 50, 255));
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid; 

}


QComboBox::down-arrow
{
    image: url(://arrow-down.png);
    width: 8px;
    height: 8px;
}


/*-----QSpinBox &amp; QDateTimeEdit-----*/
QSpinBox,
QDateTimeEdit 
{
    background-color: #131313;
	color : #eee;
	border: 1px solid #343434;
	padding: 3px;
	padding-left: 5px;
    border-radius : 2px;

}


QSpinBox::up-button, 
QDateTimeEdit::up-button
{
	border-top-right-radius:2px;
	background-color: #777777;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::up-button:hover, 
QDateTimeEdit::up-button:hover
{
	background-color: #585858;

}


QSpinBox::up-button:pressed, 
QDateTimeEdit::up-button:pressed
{
	background-color: #252525;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::up-arrow,
QDateTimeEdit::up-arrow
{
    image: url(://arrow-up.png);
    width: 7px;
    height: 7px;

}


QSpinBox::down-button, 
QDateTimeEdit::down-button
{
	border-bottom-right-radius:2px;
	background-color: #777777;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::down-button:hover, 
QDateTimeEdit::down-button:hover
{
	background-color: #585858;

}


QSpinBox::down-button:pressed, 
QDateTimeEdit::down-button:pressed
{
	background-color: #252525;
    width: 16px; 
    border-width: 1px;

}


QSpinBox::down-arrow,
QDateTimeEdit::down-arrow
{
    image: url(://arrow-down.png);
    width: 7px;
    height: 7px;

}

/*-----QGroupBox-----*/
QGroupBox 
{
    border: 1px solid;
    border-color: #666666;
	border-radius: 5px;
    margin-top: 20px;

}


QGroupBox::title  
{
    background-color: transparent;
    color: #eee;
    subcontrol-origin: margin;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


/*PREDEFINED*/
#toolButton{
 border:none
}

#frTop1{
 background-color:#808080
}

#frTop2{
 background-color:#808080
}

#lblUser {
	background-color: transparent;
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>20</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 24pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>SALES REPORT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>12</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/icons/957216-ui-interface/png/094-agenda.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbTag_5">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>59</y>
      <width>571</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="title">
     <string>OPTION</string>
    </property>
    <widget class="QLabel" name="label_59">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>27</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>DATE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>28</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Today</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yesterday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>This Week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last Week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>This Month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last Month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last 30 Days</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Last 90 Days</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Year To Date</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom...</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_60">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>52</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>FROM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_61">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>76</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>TO</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDateEdit" name="dtSortFrom">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>53</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateEdit" name="dtSortTo">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>78</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frTop1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>27</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblDebit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1677790</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);
background-color: rgb(52, 52, 52);</string>
      </property>
      <property name="text">
       <string>TOTAL CREDIT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTotCredit">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>35</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frTop2">
     <property name="geometry">
      <rect>
       <x>419</x>
       <y>28</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblDebit_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1677790</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);
background-color: rgb(52, 52, 52);</string>
      </property>
      <property name="text">
       <string>TOTAL DEBIT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTotDebit">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>36</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QToolButton" name="btnRefresh">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>30</y>
       <width>51</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">

</string>
     </property>
     <property name="text">
      <string>REFRESH</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>recources/icons/957216-ui-interface/png/060-refresh.png</normaloff>recources/icons/957216-ui-interface/png/060-refresh.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbCustList_3">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>169</y>
      <width>571</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="title">
     <string>Record(s) Found</string>
    </property>
    <widget class="QTableWidget" name="tblLedger">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>551</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Century Gothic&quot;;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>DATE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TRN #</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CREDIT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DEBIT</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnRefresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayReport()</slot>
 </slots>
</ui>
